library/xml-model/src/main/java/org/cru/godtools/xml/model/tract/TractPage.kt

```diff
@@ -73,6 +73,11 @@ class TractPage : BaseModel, Styles {
     @get:ColorInt
     val cardTextColor get() = _cardTextColor ?: textColor
 
+    @ColorInt
+    private val _cardBackgroundColor: Int?
+    @get:ColorInt
+    val cardBackgroundColor get() = _cardBackgroundColor ?: manifest.cardBackgroundColor
```

Comment:
I'm actually glad you implemented this here, rethinking the one long elvis operator chain, I was going to have you refactor to utilizing this.